src/api/apis/BoardGameGeekAPI.ts

```diff
@@ -18,6 +18,11 @@ export class BoardGameGeekAPI extends APIModel {
 		this.types = [MediaType.BoardGame];
 	}
 
+	processImageLink(boardgameElement: { querySelector: (arg0: string) => { (): any; new (): any; textContent: any } }) {
```

Comment:
There should be a better type for this, no? I am guessing `boardgame` has a type that can be used here.